packages/fxa-settings/src/pages/Signup/SignupConfirmed/index.test.tsx

```diff
@@ -23,7 +25,7 @@ describe('SignupConfirmed', () => {
     render(<SignupConfirmed />);
     const ftlMsgMock = screen.getAllByTestId('ftlmsg-mock')[1];
     testL10n(ftlMsgMock, bundle, {
-      serviceName: 'account settings',
+      serviceName: MozServices.Default,
```

Comment:
Hmm... definitely not for this PR, but long-term I think we may want to refactor these l10n strings so that it localizes `account settings` instead of holding it as a var.

Reply:
Actually I think we already did that -- we _should_ be able to remove var?

Reply:
I had a surprised Pikachu moment there reading this but I think we've done this right! 😅 Search for `headingWithDefaultServiceFtlId` and what those IDs are tied to, I'm seeing us properly sending `account settings` in as-is rather than tying it to a variable.

Reply:
Agreed! Am I incorrect in thinking we can remove this var then for `testl10n`? Like if the component doesn't get a serviceName value, then testL10n shouldn't need one either?

Reply:
Yeah I believe we can for this one. I could also change it to a different service but removing it is easier.